### ISSUE TITLE
import  OnMyLocationClickListener fails, cannot resolve

### ISSUE THREAD

**Issue by simonbuehler** [googlemaps/android-samples#70]
hi,

tried the sample but the line 
`import com.google.android.gms.maps.GoogleMap.OnMyLocationClickListener;` gives me a unresolved error, how can i fix it and what causes it?

**Comment by simonbuehler** [comment]
it seems the latest version is needed but not available in android studio:

Failed to resolve: com.google.android.gms:play-services:11.4.0

**Comment by HadiAbu** [comment]
just deleted the class MyLocationClickListener, i guess it's not supported.

and in the gradle file:

use this dependency:
    compile 'com.google.android.gms:play-services-maps:11.0.4'
instead of this:
    compile 'com.google.android.gms:play-services-maps:11.4.0'

(it's version number: 11.0.4 and not 11.4.0)

**Comment by spitfire2903** [comment]
You can solve this putting GoogleMaven in build.gradle
```
allprojects {
    repositories {
        mavenLocal()
        jcenter()
        maven {
            url 'https://maven.google.com/'
            name 'Google'
        }
    }
}
```

**Comment by katherineallen** [comment]
This has been fixed in pull request #79
